docs/backend/installation.md

```diff
@@ -0,0 +1,128 @@
+# Instalation
+
+<div class="Alert Alert--warning">
+
+This manual is prepared for the developer installation
+
+</div>
+
+## Requirements
+---
+
+| Name                  | Version                           |
+|-----------------------|-----------------------------------|
+| Language              | Php >= 7.2                        |
+| Database              | Posgress >= 10                    |
+| Message Broker*       | RabbitMQ >= 3.6.10                |  
```

Comment:
RabbitMQ is optional

Reply:
that  information are written bellow table